runbotics/runbotics-scheduler/src/scheduler-database/credential/credential.service.ts

```diff
@@ -74,13 +75,24 @@ export class CredentialService {
   }
 
   async findAllAccessible(tenantId: string, user: IUser) {
-    const accessibleCollections = await this.collectionService.findAllAccessible(user);
+    const accessibleCollections = await this.collectionService.findAllAccessibleForCredentials(user);
 
     if (accessibleCollections.length === 0) {
       throw new NotFoundException('Could not find any accessible collections');
     }
 
-    const credentials = accessibleCollections.map((collection) => collection.credentials).flat();
+    const credentials = accessibleCollections.map((collection) => collection.credentials)
```

Comment:
it's always safer to provide param names yourself instead of using spread operator

Reply:
That is true, I think there is a need for DTO mapper in this service and this should be added as a refactor for all of those services